README.md

```diff
@@ -9,6 +9,7 @@ Some rails specific options.
 
 ```ruby
 set :rails_env, 'staging'                  # If the environment differs from the stage name
+set :rails_assets_groups, nil              # RAILS_GROUPS env value for the assets:precompile task
```

Comment:
Maybe we should name just `rails_groups`? May be confusing to have `rails_assets_groups` setting that corresponds to `RAILS_GROUPS` ENV variable.